src/tools/lottie2gif/lottie2gif.cpp

```diff
@@ -25,7 +25,11 @@
 #include <vector>
 #include <thorvg.h>
 #ifdef _WIN32
+    #ifndef WIN32_LEAN_AND_MEAN
```

Comment:
In tools, these are just optional, not mandatory.
probably, we can remove them add them in meson 
or at least just single line of  `#define WIN32_LEAN_AND_MEAN` without any ancillary code.